.travis.yml

```diff
@@ -1,18 +1,23 @@
 dist: trusty
 
 os:
-  - linux
+- linux
 
 language: go
 
 go:
-  - 1.7
-  - tip
+- 1.7
+- tip
 
 before_install:
-  - go get github.com/Masterminds/glide
+- go get github.com/Masterminds/glide
+- go get github.com/mattn/goveralls
+- go get -u github.com/alecthomas/gometalinter
```

Comment:
let's drop the `-u` as it shouldn't have any effect on travis (new container all the time)